tests/test_embedding.py

```diff
@@ -24,7 +24,7 @@ def test_embedding_dburl(self):
             assert dbpath.exists()
 
     def test_arguments(self):
-        with TemporaryDirectory() as t:
+        with TemporaryDirectory() as t, mock.patch("warnings.warn") as w:
```

Comment:
You could have used https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings instead of mock, which looks like it would make it straightforward to assert that the expected warning was shown